volunteers/forms.py

```diff
@@ -1,7 +1,177 @@
-from .models import *
-from django.forms import ModelForm
+from .models import Volunteer, Interest, Language, Event, OrganizationAffiliation, Training, Role
+from django import forms
+
 
 class VolunteerForm(forms.ModelForm):
+    last_name = forms.CharField(
+        max_length=30,
+        widget=forms.TextInput(
+            attrs={
+                "class": "form-control"
+            }
+        )
+    )
+    first_name = forms.CharField(
+        max_length=30,
+        widget=forms.TextInput(
+            attrs={
+                'class': 'form-control'
+            }
+        )
+    )
+    middle_name = forms.CharField(
+        max_length=30,
+        widget=forms.TextInput(
+            attrs={
+                'class': 'form-control'
+            }
+        )
+    )
+    gender = forms.ChoiceField(
+        choices=(
+            ("M", "Male"),
+            ("F", "Female"),
+            ("O", "Others")
+        ),
+        widget=forms.Select(
+            attrs={
+                'class': 'form-control'
+            }
+        )
+    )
+    birth_date = forms.DateField(
+        widget=forms.DateInput(
+            attrs={
+                'class': 'form-control',
+                'placeholder': "Month Day Year"
+            }
+        )
+    )
+    # for birth dates, i would go with DateField
+    physical_limitations = forms.ChoiceField(
+        choices=(
+                ("Y", "Yes"),
+                ("N", "No")
+        ),
+        widget=forms.RadioSelect(
+            attrs={
+                'class': 'radio'
+            }
+        )
+    )
+    physical_explanation = forms.CharField(
+        widget=forms.Textarea(
```

Comment:
Text inport just gives a small box for someone to type in... it it the equival of the html <input type"text">
I used textareas so that in the post we can use widgets so the users can proper edit operations, we will plugins to enhance the text area functionality. If run your development server you should see it big text area input for textareas.